src/PlatformEngines.jl

```diff
@@ -855,6 +855,62 @@ telemetry_notice(server::AbstractString=pkg_server()) = """
     LEGAL NOTICE: package operations send anonymous data about your system to $server (your current package server), including the operating system and Julia versions you are using, and a random client UUID. Running `Pkg.telemetryinfo()` will show exactly what data is sent. See https://julialang.org/legal/data/ for more details about what this data is used for, how long it is retained, and how to opt out of sending it.
```

Comment:
It would be nice to update `telemetry_notice` to give information about `telemetry` command.

Reply:
Good idea, I've added something, even if the verbiage is not ideal.

Reply:
@staticfloat In that case, can you add "fixes #1898" to the description of this PR?